static/tabs/setup.html

```diff
@@ -0,0 +1,85 @@
+<div
+    class="tab-pane fade{% if not configure %} show active{% endif %}"
+    id="setup"
+    role="tabpanel"
+    aria-labelledby="setup-tab"
+>
+    <div class="container">
+        <div class="row g-3">
+            <div class="col-md-6">
+                <form
+                    action="{{ url_for('app_api_ankerctl_config_upload') }}"
+                    method="POST"
+                    enctype="multipart/form-data"
+                    class="mb-3"
+                    id="config-upload-form"
+                >
+                    <div class="card">
+                        <div class="card-header">
+                            <label for="loginFile" class="fs-4">
+                                Upload AnkerMake Login File
```

Comment:
We should probably wrap this in `<h4>` or remove the `<h4>` wrapper around the other section heads so that the font-types are the same. 

![image](https://github.com/Ankermgmt/ankermake-m5-protocol/assets/3013565/55bc1189-b953-4f28-8c37-500b8c2bc45b)

Reply:
Good point - I actively removed the `<h4>` here, since neither `<label>` or `<h4>` is allowed to nest inside the other.

I cleaned this up to use `.fs-4` in all 3 places, and removed all `<h4>` tags on this page.

:white_check_mark: